apps/test/netsim/NetSimLogEntry.js

```diff
@@ -0,0 +1,111 @@
+var testUtils = require('../util/testUtils');
+var assert = testUtils.assert;
+var assertEqual = testUtils.assertEqual;
+var assertWithinRange = testUtils.assertWithinRange;
+var assertOwnProperty = testUtils.assertOwnProperty;
+var netsimTestUtils = require('../util/netsimTestUtils');
+var fauxShard = netsimTestUtils.fauxShard;
+var assertTableSize = netsimTestUtils.assertTableSize;
+
+var NetSimLogEntry = testUtils.requireWithGlobalsCheckBuildFolder('netsim/NetSimLogEntry');
+
+describe("NetSimLogEntry", function () {
+  var testShard;
+
+  beforeEach(function () {
+    testShard = fauxShard();
```

Comment:
fauxShard -> fakeShard

I'm suggesting this rename because "fake" has a specific meaning in as a test double http://en.wikipedia.org/wiki/Test_double whereas "faux", to my knowledge, does not.